tests/parser/tests/cases/should_fail/destructure_onto_void/case.hash

```diff
@@ -0,0 +1,2 @@
+// This doesn't make much sense semantically either.
+let (,) = k;
```

Comment:
Why should this fail? Can we not match on empty tuples?

Reply:
Yes, this can be added as support in the parser although it doesn't make much sense to do this.

Reply:
Well, you're right it doesn't make sense now, but it might make sense if we ever have type unions. I guess for now it can be left like this.